src/components/HomePage/ContactSection.spec.js

```diff
@@ -0,0 +1,49 @@
+import React from "react";
+import ContactSection from "./ContactSection";
+import { render } from "@testing-library/react";
+
+describe("ContactSection", () => {
+    describe("render", () => {
+        it("check mail button link", () => {
+            const { queryByTitle } = render(<ContactSection />);
+            const btn = queryByTitle("email");
+            expect(btn.getAttribute("href").match("mailto:ni@aefeup.pt"));
```

Comment:
You are using regex to match the string, but you can use the string comparison directly, which is probably more performant.

Additionally, you are not "expecting" anything, since you need a matcher after the expect (https://jestjs.io/docs/en/expect#expectvalue)

The tests (and this goes for all the tests you have here -I won't repeat this comment) should follow something like this:

```js
expect(btn.getAttribute("href")).toBe("mailto:ni@aefeup.pt");
```